.github/workflows/build-and-test.yml

```diff
@@ -96,6 +92,8 @@ jobs:
     needs: build-web-app-reporter
     runs-on: ubuntu-22.04
     steps:
+    - name: Set tool version environment variables
```

Comment:
Does this commit actually align (change) any tool version?

Reply:
Yes, the ScanCode version.

Reply:
But weird that `.versions` is not found, as this step `needs: build-web-app-reporter`, so a checkout should be available...

Reply:
Nevermind, different jobs do not share the workspace.